edx_exams/apps/core/models.py

```diff
@@ -9,6 +10,8 @@
 
 
 class User(AbstractUser):
+    # todo: should I update this docstring to remove username and email address and change to
```

Comment:
I wasn't sure if this docstring is meant to include username and email address or if this is outdated. If so, I can update to reflect current user model? I do see that these fields are in the AbstractUser so maybe this is why it's here as PII?

Reply:
this docstring is all PII retirement related so I don't think there is anything to add from this work, anonymous ID is not really PII